### DISCUSSION TITLE
博客升级至 v2.0

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#386] [Announcements]
# 博客升级至 v2.0

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/2018/12/03/bump-to-version-2.0

**Comment by zflovecoding** [comment]
xuanwo哥哥的博客是什么主题呀？

**Reply by Xuanwo** [reply]
是照着一个前端工程师的博客手糊的～，这主要用到了 [tailwindcss](https://tailwindcss.com/)
